lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

```
@@ -1,4 +1,5 @@
 startup.error=An error occurred while loading this map. Some necessary resources may temporarily be unavailable. Please try again later.
+startup.user_defined_js=There's some user defined javascript scripts, that can be the cause of this message, try disabling them.
```

```suggestion
startup.user_defined_js=There's some user defined JavaScript scripts, that can be the cause of this message, try disabling them.
```

According to Wikipédia https://fr.wikipedia.org/wiki/JavaScript